changelog/fix_fix_pollution_between_tests.md

```diff
@@ -0,0 +1 @@
+* [#10992](https://github.com/rubocop/rubocop/pull/10992): Fix pollution between tests. ([@tdeo][])
```

Comment:
There is no user impact, so no changelog is required.

Reply:
Indeed, I think I went through the checklist a bit fast